starter-core/src/test/groovy/io/micronaut/starter/feature/camunda/ZeebeSpec.groovy

```diff
@@ -23,11 +23,12 @@ import io.micronaut.starter.build.dependencies.Scope
 import io.micronaut.starter.feature.CommunityFeatureValidator
 import io.micronaut.starter.fixture.CommandOutputFixture
 import io.micronaut.starter.options.BuildTool
+import spock.lang.PendingFeature
 import spock.lang.Requires
 
-@Requires({ CommunityFeatureValidator.ENABLE_COMMUNITY_FEATURES })
 class ZeebeSpec extends ApplicationContextSpec implements CommandOutputFixture {
 
+    @PendingFeature(reason = "camunda community features do not support Micronaut Framework 4 yet")
```

Comment:
agora features below, we `@Ignore` the whole spec with a message

I kinda prefer that to this `@PendingFeature` per test method

But that's an opinion/consistency thing...

Reply:
in the unit tests I used @PendingFeature

However, in the functional tests I intentionally used `@Ignored` because I don't want the functional test to run since they are time consuming.

Reply:
👍